pkg/runtime/runtime.go

```diff
@@ -147,18 +152,26 @@ func Download(version *semver.Version, name string) (*Runtime, error) {
 		runtime.Spec.Version = semver.MustParse("v99.99.99")
 	}
 
+	filteredComponets := make([]AppDef, 0)
 	for i := range runtime.Spec.Components {
-		if runtime.Spec.Components[0].Type != "kustomize" {
+		if !shouludInstallFeature(installFeatures, runtime.Spec.Components[i].Feature) {
+			continue
+		}
```

Comment:
filter components by installFeatures - will end up saved in the `bootstrap/<runtimeName>.yaml` file in the install repo at the end